spec/classes/foreman_proxy__plugin__salt_spec.rb

```diff
@@ -36,6 +37,7 @@
           should contain_file('/etc/foreman-proxy/settings.d/salt.yml').
             with_content(/:salt_command_user: example/).
             with_content(/:autosign_file: \/etc\/salt\/example.conf/).
+            with_content(/:autosign_key_file: \/var\/lib\/foreman-proxy\/salt\/grains\/autosign_key/).
```

Comment:
It's fine to keep it like this for now but for future reference, you can use `%r{something/with/slashes}`. I learned this after implementing the lines around this.

Reply:
I've created another PR, could you have a look at [this](https://github.com/theforeman/puppet-foreman_proxy/pull/718) maybe?

Reply:
Thanks. I've merged that. Could you rebase on it to resolve the merge conflicts?